### PR TITLE
Update Readme with dependent services

### PR DESCRIPTION
**Pull request by milan-tom**
I believe that this covers all of the dependent services required to run this project smoothly. Please feel free to suggest any more that need to be added.

<a href="https://gitpod.io/#https://github.com/Xcov19/covidX/pull/64"></a>

---
This change is [Reviewable](https://reviewable.io/reviews/xcov19/covidx/64)

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -17,6 +17,13 @@ We are an open community of volunteers without a commercial purpose. We believe
 #### Python Version
 covidX will be run on python *3.7.6* and *3.8.5*
 
+#### Pre-requisite Services
```
Comment (codecakes) [inline Readme.md:20]: if you look at .env file and the dependent services, you will realize this is not enough.
Look at the Applications being installed in `settings.py` again; 
In addition to that, We also need ALGOLIA api key, can you think of other dependencies here?

**Review by codecakes — CHANGES_REQUESTED** [review]
some dependencies are missing

File: Readme.md (hunk 1 above)
Comment (milan-tom) [reply Readme.md:20]: I thought that new developers were provided with the Algolia api key by the team leads. So, are you saying I should give them instructions on how to obtain this themselves?

Comment (codecakes) [reply Readme.md:20]: no. the objective of this description is for anyone who wants to self-host this repository to understand what all services are needed in order to spin up their own instance.

Comment (milan-tom) [reply Readme.md:20]: That makes a lot more sense now. So, they will need everything.

Comment (codecakes) [reply Readme.md:20]: yes, you need to tell them what services one would need to spin this up

**Review by codecakes — APPROVED** [review]
pls merge @milan-tom

**Pull request merged**